### ISSUE TITLE
= to CloudFlare Workers?

### ISSUE THREAD

**Issue by tomByrer** [lagonapp/serverless#56]
If so, might want to join [WinterCG](https://wintercg.org/).

I was fairly [hot on CFW](https://github.com/lukeed/awesome-cloudflare-workers) ~2 years ago, but got spooked with vendor lock-in, Google tie-ins, & politically vocal engineers.

This might be a better route than [Just-JS](https://just.billywhizz.io/blog/on-javascript-performance-01/) & [Unikernal/MicroVM](https://nanovms.com/).  I'm not sure if this could help my Puppeteer/Playwright project, but otherwise this looks promising.

**Comment by QuiiBz** [comment]
Thanks! To be clearer, the goal of Lagon is to provide an open source alternative to CF Workers and Deno Deploy, with the same Web APIs (some of them are already implemented at https://github.com/lagonapp/serverless/tree/main/packages/runtime/src/runtime)

WinterCG is something that I have in mind, but Lagon is still in its very early stage. I'll try to reach them and will keep this issue updated.

**Comment by tomByrer** [comment]
I've interacted with Evan on CFWorkers Discord @ open-sourcing their code (they blogged they plan to), so likely he'd be a good POC.
https://github.com/evanderkoogh

**Comment by QuiiBz** [comment]
Yeah I know they want to open source it, but not sure if it's only the runtime or the whole platform?

Here is Cloudflare's blog post about open-sourcing Workers, if anyone reading this wasn't aware of it: https://blog.cloudflare.com/workers-open-source-announcement/

**Comment by moderation** [comment]
Workerd is now open source - [Introducing workerd: the Open Source Workers runtime](https://blog.cloudflare.com/workerd-open-source-workers-runtime/) and launched with a large investment fund - [Leading venture capital firms to provide up to $1.25 BILLION to back startups built on Cloudflare Workers](https://blog.cloudflare.com/workers-launchpad/)
